djstripe/fields.py

```diff
@@ -45,6 +48,11 @@ class StripeFieldMixin(object):
     deprecated = False
 
     def __init__(self, *args, **kwargs):
+        """Assign class instance variables based on kwargs.
```

Comment:
pydocstyle doesn't mind it being on the first line in these cases but ya we should just try to keep it one style